### PR TITLE
added websocket server to keep up with number of people who joined jits room #36

### PR DESCRIPTION
**Pull request by soichih**
Adding the *probable* number of people in the Jitsu room. I've created a server side components running on my dev machine `wss://dev1.soichi.us/ohbm2020` which counts the number of the most recent IP address for each poster. I keep the IP address for 15 minutes, and broadcast update events to all clients. 

I might have to adjust the CORS once it goes live.

### CODE REVIEW

**Comment by yarikoptic** [comment]
woooohhooo!!! Will test locally now, but I guess CORR would kick in. I think I will be on 96.120.70.1

**Comment by yarikoptic** [comment]
hm, count does increase when I click!  could we try in some room (gimme number) and see if total count goes up?

**Comment by effigies** [comment]
@yarikoptic I'll join bids-derivatives

**Comment by effigies** [comment]
(Poster 1895)

**Comment by soichih** [comment]
Cool. I see your connection coming through.  I am on 1895 now.

File: assets/js/main.js [hunk 1]
```diff
@@ -474,6 +474,34 @@ function directory(jQuery) {
       {data: "videochat", title: "Video Chat", className: "dt-left", width: "18%"},
       {data: "pdf", title: "PDF", className: "dt-left", width: "8%"},
     ],
+
+    columnDefs: [
+        {
+            render(data, type, row) {
+                return `
+                    <button type="button" onclick="openJit('https://meet.jit.si/ohbm2020-${row.number}', ${row.number})">
+                        Open
```
Comment (yarikoptic) [inline assets/js/main.js:484]: Could we replace `Open` with the text of the url like it was before with a simple link?  Some of them "non-standard" so visualizing that is of added information.

**Comment by yarikoptic** [comment]
I do see 1 for 1895!!!!! joining

File: assets/js/main.js (hunk 1 above)
Comment (soichih) [reply assets/js/main.js:484]: done

**Comment by soichih** [comment]
I've updated so that it now detect someone closing the Jitsu window properly and de-crement the counter accordingly. It also logs open/close activities

**Comment by yarikoptic** [comment]
@sergeyplis also suggested to just add a dedicated column with # of participants, then we could sort by it!

**Comment by yarikoptic** [comment]
ok, it is too exciting to hold off on it.  I am merging, everything else could be improved after!

**Comment by soichih** [comment]
Sorry, I've been in a bunch of meetings this afternoon. I can work on the dedicated column - great idea!

**Comment by yarikoptic** [comment]
no problem! thanks for getting back to it!

**Pull request merged**